doc/source/cluster/kubernetes/getting-started/raycluster-quick-start.md

```diff
@@ -124,24 +124,22 @@ Now that we have the name of the service, we can use port-forwarding to access t
 ```sh
 # Execute this in a separate shell.
 kubectl port-forward service/raycluster-kuberay-head-svc 8265:8265
-
-# Visit ${YOUR_IP}:8265 in your browser for the Dashboard (e.g. 127.0.0.1:8265)
 ```
 
-Note: We use port-forwarding in this guide as a simple way to experiment with a RayCluster's services. For production use-cases, you would typically either 
-- Access the service from within the Kubernetes cluster or
-- Use an ingress controller to expose the service outside the cluster.
-
-See the {ref}`networking notes <kuberay-networking>` for details.
-
 Now that we have access to the Dashboard port, we can submit jobs to the RayCluster:
 
 ```sh
 # The following job's logs will show the Ray cluster's total resource capacity, including 2 CPUs.
 ray job submit --address http://localhost:8265 -- python -c "import ray; ray.init(); print(ray.cluster_resources())"
 ```
 
-## Step 5: Cleanup
+## Step 5: Access the Ray Dashboard and submit a job
+
+Visit `${YOUR_IP}:8265` in your browser for the Dashboard (e.g. `127.0.0.1:8265`)
```

Comment:
the step title says "submit a job", where this instruction and the figure only says visit the dashboard? seems that job submission is actually in `Step 4`?

other than being able to visit the dashboard, is there something else user should expect to see?

Reply:
Updated